docs/differential-analysis.rst

```diff
@@ -0,0 +1,165 @@
+Differential Analysis
+=====================
+
+Differential Analysis provides the ability to compare and understand the changes between two
+`Live Query <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/latest/livequery-api/>`_ runs.
+The differential is calculated based on point-in-time snapshots. These features answer the question, “What changed on endpoints, and when?”.
+
+Overview
+--------
+This guide follows the steps for comparing two “point-in-time snapshots” of endpoints using a few different options and downloading the results using the Differential object.
+This example aims to understand what Firefox add-ons were added or removed between the two Live Query snapshot intervals.
+
+**1. Prerequisites**
+
+To perform a Differential Analysis, create the "point-in-time" snapshots of your endpoints with Live Query or use existing ones.
+You can find a step-by-step Live Query API guide `here <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/latest/livequery-api/#quick-start>`_ and
+a version for the CBC Python SDK `here <https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/live-query/>`_.
+The example Live Query runs look for added or removed Firefox add-ons.
+
+**2. Query Comparison**
+
+Start a Query Comparison with the id's you received from step 1. In some cases, the ``older_run_id`` is not required.
```

Comment:
What happens if you don't supply `older_run_id`? Does it just assume "previous run to the run specified by `newer_run_id` "? This may be covered in the Dev Network docs, but it would be nice to see mention of it here.

Reply:
It is covered in the dev net, but nice catch - adding it.
How it works - If it is a recurring Live Query run, it will automatically get previous to the supplied one. If you supply just a single run query id, the API will throw a specific error.